x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/InternalRealms.java

```diff
@@ -52,15 +54,24 @@
      */
     private static final Set<String> XPACK_TYPES = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(
             NativeRealmSettings.TYPE, FileRealmSettings.TYPE, LdapRealmSettings.AD_TYPE, LdapRealmSettings.LDAP_TYPE, PkiRealmSettings.TYPE,
-            SamlRealmSettings.TYPE
+            SamlRealmSettings.TYPE, KerberosRealmSettings.TYPE
     )));
 
+    /**
+     * The list of all realm types, which are those that have extensive interaction
+     * with third party sources
+     */
+    private static final Set<String> TYPES_DEPEND_ON_THIRD_PARTY_SOURCES = new HashSet<>();
+    static {
+        Collections.addAll(TYPES_DEPEND_ON_THIRD_PARTY_SOURCES, SamlRealmSettings.TYPE, KerberosRealmSettings.TYPE);
+    }
```

Comment:
Why use a static block to initialise this?
`Sets.newHashSet` can turn this into a 1 liner, and then you can wrap it in `unmodifiableSet`